fixtures/exported/10-runtimes.neon

```diff
@@ -0,0 +1,61 @@
+App\Model\Entity\RuntimeEnvironment:
+	c-gcc-linux:
+		__construct:
+			- c-gcc-linux
+			- "C (GCC)"
+			- C
+			- "[c, h]"
+			- GNU/Linux
+			- C compiled with GCC
+			- "- name: \"source-files\"\r\n  type: \"file[]\"\r\n  value: \"*.{c,h}\""
```

Comment:
Couldn't this be done without \r\n?

Reply:
It is generated automatically...

Reply:
And can it be generated without \r\n? :P

Reply:
Don't know, don't care 😀

Reply:
Funny thing is, it was generated at our production server with CentOS